bullet_train-themes-light/app/views/showcase/previews/partials/_decision_line.html.erb

```diff
@@ -0,0 +1,7 @@
+<% showcase.sample "Basic" do %>
+  <%= render "shared/decision_line", label: 'else' %>
+<% end %>
+
+<% showcase.options do |o| %>
+  <% o.optional :label, "Label for the decision line", default: t('global.or') %>
```

Comment:
Maybe we should do the default as a string, so people know how it's translated? And what key they'd need to replace?

```suggestion
  <% o.optional :label, "Label for the decision line", default: "t('global.or')" %>
```